wit/keyvalue.wit

```diff
@@ -0,0 +1,26 @@
+package wasmcloud:keyvalue@0.1.0
+
+/// An interface for reading and writing keys and their respective values
+interface readwrite {
+    get: func(key: string) -> result<option<string>, error>
+
+	
+	set: func(key: string, value: string) -> result<_, error>
+
+	
+	delete: func(key: key) -> result<_, error>
+
+	
+	exists: func(key: key) -> result<bool, error>
```

Comment:
is this weird tab/spaces?

Is there an official recommendation with 2/4 space indentation for wit?

Reply:
I have no idea on either count.

Reply:
Yeah the 2 empty lines in between each line is really throwing me for some reason 😆